conans/test/integration/command_v2/test_info_build_order.py

```diff
@@ -526,3 +532,26 @@ def export(self):
     c.run("graph build-order --requires=dep/0.1 --format=json", assert_error=True)
     assert "ImportError" in c.out
     assert "It is possible that this recipe is not Conan 2.0 ready" in c.out
+
+
+class TestBuildOrderReduce:
+    def test_reduce(self):
+        c = TestClient()
+        c.save({"liba/conanfile.py": GenConanfile("liba", "0.1"),
+                "libb/conanfile.py": GenConanfile("libb", "0.1").with_requires("liba/0.1"),
+                "libc/conanfile.py": GenConanfile("libc", "0.1").with_requires("libb/0.1"),
+                "consumer/conanfile.txt": "[requires]\nlibc/0.1"})
+        c.run("create liba")
+        c.run("create libb")
+        c.run("create libc")
+        c.run("remove liba:* -c")
+        c.run("remove libc:* -c")
+        c.run("graph build-order consumer --build=missing --reduce --format=json")
+        bo_json = json.loads(c.stdout)
```

Comment:
Would it be interesting to see the whole JSON output? At least, we could show it in the docstrings to appreciate the difference.

Reply:
The JSON output is very different for both recipes/configuration orders, so adding the whole JSON adds around +100 lines of code that is mostly already covered above, the only difference is the ``depends`` that changes